src/database/config.js

```diff
@@ -0,0 +1,31 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+const config = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DEV,
+    host: process.env.DB_HOST,
+    port: process.env.PORT,
+    dialect: 'postgres',
+  },
+  test: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_TEST,
+    host: process.env.DB_HOST,
+    port: process.env.PORT,
+    dialect: 'postgres',
+  },
+  production: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE,
+    host: process.env.DB_HOST,
+    port: process.env.PORT,
+    dialect: 'postgres',
+  }
+};
+
+export default config;
```

Comment:
There is a lot of repetitive code here. Please, refactor. Also, since we are using a DB URL on Heroku there is no need for production config you did. instead, a URL would be used.

Reply:
Thank you. I have just implemented this.